features/gba-dev/rni-gba.feature

```diff
@@ -64,7 +64,6 @@ Functionaliteit: GBA : RNI-deelnemer voor geleverde gegevens
       | voornamen | Peter  |
       En heeft de persoon de volgende 'verblijfplaats' gegevens
       | naam                    | waarde                   |
-      | type                    | VerblijfplaatsBuitenland |
       | verblijfadres.land.code | 5010                     |
```

Comment:
Verblijfplaats kent geen verblijfadres in de GBA. Dat was deel van de fout waar mijn opmerking over ging :).